models/update-article.model.js

```diff
@@ -0,0 +1,37 @@
+const db = require("../db/connection");
+
+exports.updateArticle = (article_id, articleUpdate) => {
+  if (Object.keys(articleUpdate).length === 0) {
+    return Promise.reject({
+      message: "No data provided. Cannot execute request!",
+      status: 400,
+    });
+  }
+  if (isNaN(article_id)) {
+    return Promise.reject({
+      message: "This is a bad request, invalid article id!",
+      status: 400,
+    });
+  }
+  if (isNaN(articleUpdate.inc_votes)) {
+    return Promise.reject({
+      message: "This is a bad request, invalid input!",
+      status: 404,
```

Comment:
I'd console log some of these errors and see what you're getting back. I'm not sure you need to do all of these Promise.rejects, as SQL should throw you some errors you can handle in your middleware!